src/PRo3D.Viewer/Viewer/Viewer-Utils.fs

```diff
@@ -377,6 +379,7 @@ module ViewerUtils =
                     |> Sg.trafo trafo //(Transformations.fullTrafo surf refsys)
                     |> Sg.modifySamplerState DefaultSemantic.DiffuseColorTexture samplerDescription
                     |> Sg.uniform "selected"      (isSelected) // isSelected
+                    |> Sg.uniform "useTexcoords" (AVal.constant false) //(useTexcoords) 
```

Comment:
on aardvark style, uniforms should be written in capitals. I think this would be a large consolidation pass in Pro3D (also when we fix all wrong tabs)